docs/index.html

```diff
@@ -89,6 +89,7 @@ <h2 class="title">A 1x Engineer...</h2>
           <li>May decorate their laptop and workspace in any way they like, and is respectful of others' decor (or lack thereof), too.</li>
           <li>Isn't defined by myopic Tweetstorms by clueless VCs.</li>
           <li>Doesn't ridicule entire professions within engineering, especially not when in a position of leadership.</li>
+          <li>Sometimes the love-hate relationship for this profession could be modeled as a sine wave.</li>
```

Comment:
ah does this mean that it can be up and down, so we should not let love-hate prevent our professional?